imports/components/player_list.jsx

```diff
@@ -0,0 +1,44 @@
+export class PlayerList extends React.Component {
+  render() {
+    var players = this.props.players.map(function(player) {
+      return (<PlayerListEntry key={player._id} {...player} />);
+    });
+    var logIn = null;
+    if (!this.props.loggedIn) {
+      logIn = <li>
+        <strong>Log In</strong> to join this list
+      </li>
```

Comment:
You're missing a semicolon here, and on the PlayerListEntry render method. I've been pretty happy with eslint for catching this sort of thing. I set it up to run via Circle, and it's been pretty good. There's a meteor plugin you definitely want.